schemas/com.snowplowanalytics.amp/amp_web_page/jsonschema/1-0-0

```diff
@@ -0,0 +1,20 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"self": {
+		"vendor": "com.snowplowanalytics.amp",
```

Comment:
Does this use tabs instead of spaces? It looks way wider than the other schema.

Reply:
Weird. Will fix.